### DISCUSSION TITLE
How to compile webrtcjni for aarch64

### DISCUSSION THREAD

**Discussion by hussainasad6696** [devopvoid/webrtc-java#65] [Q&A]
My application is working on ubuntu arm but not on aarch64 debian but as per mentioned it is not compatible with aarch64. is there a way to build webrtc wrapper or jni for aarch64

**Comment by devopvoid** [comment]
You can try to derive the build process from the GitHub actions workflow in the directory `.github/`. ARM and ARM64 for Linux are available with the latest release. Apple Silicon is also supported.
